eng/tools/generator/autorest/model/changelog.go

```diff
@@ -265,8 +269,10 @@ func getSignatureChangeItems(b *report.BreakingChanges) []string {
 		for _, k := range sortChangeItem(b.Structs) {
 			v := b.Structs[k]
 			for f, d := range v.Fields {
-				line := fmt.Sprintf("Type of `%s.%s` has been changed from `%s` to `%s`", k, f, d.From, d.To)
-				items = append(items, line)
+				if !strings.Contains(d.To, "any") {
```

Comment:
Could you please remove duplication of these code?

Reply:
Duplicate code means duplication with function typeToAny?

Reply:
> Duplicate code means duplication with function typeToAny?

Yes.